src/core/client/stream/tabs/Comments/FlattenedReplyList/FlattenedReplyListContainer.tsx

```diff
@@ -0,0 +1,241 @@
+
+import { CommentContainer } from "../Comment";
+import CollapsableComment from "../Comment/CollapsableComment";
+import { isPublished } from "../helpers";
+import Indent from "../Indent";
+import FlattenedReplyListViewNewMutation from "./FlattenedReplyListViewNewMutation";
+
+type FragmentVariables = Omit<
+  FlattenedReplyListContainerPaginationQueryVariables,
+  "commentID"
+>;
+
+interface Props {
+  viewer: FlattenedReplyListContainer_viewer;
+  story: FlattenedReplyListContainer_story;
+  comment: FlattenedReplyListContainer_comment;
+  settings: FlattenedReplyListContainer_settings;
+  relay: RelayPaginationProp;
+}
+
+const FlattenedReplyListContainer: FunctionComponent<Props> = ({
+  viewer,
+  story,
+  comment,
+  settings,
+  relay,
+}) => {
+  const [showAll, isLoadingShowAll] = useLoadMore(relay, 999999999);
+  const beginShowAllEvent = useViewerNetworkEvent(ShowAllRepliesEvent);
+
+  const onShowAll = useCallback(async () => {
+    const showAllEvent = beginShowAllEvent({ commentID: comment.id });
+    try {
+      await showAll();
+      showAllEvent.success();
+    } catch (error) {
+      showAllEvent.error({ message: error.message, code: error.code });
+      // eslint-disable-next-line no-console
+      console.error(error);
+    }
+  }, [beginShowAllEvent, comment.id, showAll]);
+
+  const viewNew = useMutation(FlattenedReplyListViewNewMutation);
+  const onViewNew = useCallback(() => {
+    void viewNew({ commentID: comment.id, storyID: story.id });
+  }, [comment.id, story.id, viewNew]);
+
+  const viewNewCount =
+    (comment.replies.viewNewEdges && comment.replies.viewNewEdges.length) || 0;
+
+  if (
+    comment.replies === null ||
+    (comment.replies.edges.length === 0 && viewNewCount === 0)
+  ) {
+    return null;
+  }
+
+  const comments =
+    comment.lastViewerAction && !isPublished(comment.status)
+      ? []
+      : comment.replies?.edges.map((edge) => edge.node);
+
+  return (
+    <>
+      <HorizontalGutter
+        id={`coral-comments-replyList-log--${comment.id}`}
+        data-testid={`commentReplyList-${comment.id}`}
+        role="log"
+      >
+        {comments.map((c) => (
+          <HorizontalGutter key={c.id}>
+            <CollapsableComment>
+              {({ collapsed, toggleCollapsed }) => (
+                <CommentContainer
+                  key={comment.id}
+                  ancestorID={comment.id}
+                  viewer={viewer}
+                  story={story}
+                  comment={c}
+                  settings={settings}
+                  indentLevel={4}
+                  collapsed={collapsed}
+                  toggleCollapsed={toggleCollapsed}
+                />
+              )}
+            </CollapsableComment>
+          </HorizontalGutter>
+        ))}
+        {relay.hasMore() && (
+          <Indent level={4} noBorder>
```

Comment:
I think i'd be very useful to extract the const `4` to a variable.

Reply:
Agreed, did this in previous fixes requested by @tessalt